grakn-graql/src/main/java/ai/grakn/graql/internal/query/analytics/ClusterQueryImpl.java

```diff
@@ -55,41 +61,38 @@ public T execute() {
             return (T) Collections.emptyMap();
         }
 
-        ComputerResult result;
-
         Set<LabelId> subLabelIds = convertLabelsToIds(subLabels);
 
+        GraknVertexProgram vertexProgram;
```

Comment:
Make this `GraknVertexProgram<?> vertexProgram;` otherwise Java moves into ["raw" no generics mode](https://docs.oracle.com/javase/tutorial/java/generics/rawTypes.html) and things get weird :P

Reply:
good point